libs/ui/inputs/src/lib/url-input/url-input.component.ts

```diff
@@ -0,0 +1,33 @@
+import { ChangeDetectorRef, Component, Input, Output } from '@angular/core'
+import { CommonModule } from '@angular/common'
+import { ButtonComponent } from '../button/button.component'
+import { MatIconModule } from '@angular/material/icon'
+import { filter } from 'rxjs/operators'
+import { Subject } from 'rxjs'
+
+@Component({
+  selector: 'gn-ui-url-input',
+  templateUrl: './url-input.component.html',
+  styleUrls: ['./url-input.component.css'],
+  standalone: true,
+  imports: [CommonModule, ButtonComponent, MatIconModule],
+})
+export class UrlInputComponent {
+  @Input() value = ''
+  @Input() extraClass = ''
+  @Input() placeholder = 'https://'
+  @Input() disabled: boolean
+  rawChange = new Subject<string>()
+  @Output() valueChange = this.rawChange.pipe(filter((v) => !!v))
```

Comment:
Is this needed ? It emits falsy value sometimes ?

Reply:
Oh ok, I would have just put a `if (value.trim() === '') return` in the handleChange method.